sqlglot/dialects/presto.py

```diff
@@ -149,6 +149,8 @@ def _sequence_sql(self: generator.Generator, expression: exp.GenerateSeries) ->
 
     sql = self.func("SEQUENCE", start, end, step)
     if isinstance(expression.parent, exp.Table):
+        if expression.parent.alias:
+            expression.parent.args["alias"] = f"_u({expression.parent.alias})"
         sql = f"UNNEST({sql})"
```

Comment:
This is a bit rough!

1. I'm not sure if mutating the parent expression like this is OK
2. We probably need to ensure that the table alias name (`_u`) isn't taken